apps/condo/domains/common/utils/helpers.ts

```diff
@@ -12,6 +14,23 @@ export const formatPhone = (phone?: string): string => {
     */
 }
 
+/**
+ * Formats raw timestamp string into human readable form, depending of what year it represents:
+ * 1. For current year it returns day and month;
+ * 2. For some previous year it returns day, month and year.
+ * @param intl - i18n object from Next.js, containing `locale` prop
+ * @param dateStr - raw timestamp string to format
+ * @return {String} human readable representation of provided timestamp
+ */
+export const formatDate = (intl, dateStr?: string): string => {
```

Comment:
It's local formatDate function, not common.
As u can see we got something similar in apps/condo/domains/ticket/utils/helpers.ts

Reply:
Moved in `b41673ac`.